pkg/doc/signature/jsonld/processor.go

```diff
@@ -22,7 +22,7 @@ import (
 const (
 	format             = "application/n-quads"
 	defaultAlgorithm   = "URDNA2015"
-	handleNormalizeErr = "Error while parsing N-Quads; invalid quad. line:"
+	handleNormalizeErr = "error while parsing N-Quads; invalid quad. line:"
```

Comment:
They changed the error message format a bit, that's the only required fix in the `aries` code when switching to a newer version of `json-gold` library.

Reply:
Don't change it, this message is from json-gold, we use it to catch invalid RDF scenario. Did they change error message?

Reply:
Yes, they change it. That's why we started to get error messages. This is a tiny fix of that.